docs/user/metric-parameters.md

```diff
@@ -29,6 +31,8 @@ All metric types also support the following optional parameters:
 - `send_in_pings`: Defines which pings the metric should be sent on. If not specified, the metric is sent on the "default ping", which is the `events` ping for events and the `metrics` ping for everything else. Most metrics don't need to specify this unless they are sent on custom pings.
```

Comment:
nit: since we're here, can we add a link to the custom pings docs on the "custom pings" at the end of this sentence?